backend/src/test/java/com/team4/backend/service/InternshipOfferServiceTest.java

```diff
@@ -558,4 +559,46 @@ void shouldGetExclusiveInternshipOfferStudentViewsAlreadyAppliedMatches() {
                 .verifyComplete();
     }
 
+    @Test
+    void shouldGetAllNonValidatedOffers() {
+        //ARRANGE
+        Flux<InternshipOffer> internshipOffers = InternshipOfferMockData.getAllInternshipOffers();
```

Comment:
remove extra line and for the rest too

Reply:
to be precise im talking about the extra line at the bottom of //ARRANGE, //ACT, //ASSERT